### PR TITLE
dual_iir_configuration: fix pid parameter scaling

### PR DESCRIPTION
**Pull request by jordens**

### CODE REVIEW

File: scripts/dual_iir_configuration.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def calculate_pid_coefficients(sampling_period, args):
         [1, -2, 1]
     ]
 
-    digital_conversion = (2 * pi) / sampling_period
+    digital_conversion = 2 * pi * sampling_period
```
Comment (ryan-summers) [inline scripts/dual_iir_configuration.py:191]: I'm confused. I thought that `omega_s = 2 pi / T`.

Is this conversion type not actually `omaga_s`?

Comment (jordens) [reply scripts/dual_iir_configuration.py:191]: In the definitions of the primer, `kd*2*pi` is the D gain at the sample frequency. Or `ki/tau` is the I gain "slope". Then
`kd*2*pi*f/fs` and `ki/(2*pi*f/fs)` are gains at f.

Comment (ryan-summers) [reply scripts/dual_iir_configuration.py:191]: Ah gotcha. That makes more sense, thanks for the clarification. This is a scaling factor to convert some given `f` input.

**Pull request merged**